dvc/stage.py

```diff
@@ -598,6 +611,12 @@ def load(repo, fname):
         Stage.validate(d, fname=os.path.relpath(fname))
         path = os.path.abspath(fname)
 
+        checksum = {}
+        for k in modchecksum.CHECKSUM_MAP.keys():
+            v = d.get(k)
+            if v:
+                checksum[k] = v
+
```

Comment:
```python
checksum = {k: d[k] for k in modchecksum.CHECKSUM_MAP if d.get(k)}

# Or if you don't expect falsy values in `d`:
checksum = {k: d[k] for k in modchecksum.CHECKSUM_MAP if k in d}
```

Reply:
Or using [funcy](https://funcy.readthedocs.io/en/stable/):
```python
from funcy import project
checksum = project(d, modchecksum.CHECKSUM_MAP)
```
Funcy is my lib :)